From 46ef94eb04ea03b7ef53b156439eef22e3c0a054 Mon Sep 17 00:00:00 2001
From: Trek <trek00@inbox.ru>
Date: Sun, 7 Oct 2018 17:57:54 +0200
Subject: [PATCH] Remove the shutdown dependency on udev to fix a regression of
 #791944

As udev no longer removes his control socket on exit, it is now stopped
before sendsigs, but a circular dependecy is created because cryptdisks
must be called after umountfs. Update the init scripts to remove the
dependency on udev at shutdown.

Closes: #908796
---
 debian/cryptsetup-run.cryptdisks-early.init | 2 +-
 debian/cryptsetup-run.cryptdisks.init       | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/debian/cryptsetup-run.cryptdisks-early.init b/debian/cryptsetup-run.cryptdisks-early.init
index 6498431..c027d4a 100644
--- a/debian/cryptsetup-run.cryptdisks-early.init
+++ b/debian/cryptsetup-run.cryptdisks-early.init
@@ -4,7 +4,7 @@
 # Required-Start:    checkroot
 # Required-Stop:     umountroot
 # Should-Start:      udev mdadm-raid
-# Should-Stop:       udev mdadm-raid
+# Should-Stop:       mdadm-raid
 # X-Start-Before:    lvm2
 # X-Stop-After:      lvm2 umountfs
 # X-Interactive:     true
diff --git a/debian/cryptsetup-run.cryptdisks.init b/debian/cryptsetup-run.cryptdisks.init
index 0cd4a83..17095a7 100644
--- a/debian/cryptsetup-run.cryptdisks.init
+++ b/debian/cryptsetup-run.cryptdisks.init
@@ -4,7 +4,7 @@
 # Required-Start:    checkroot cryptdisks-early
 # Required-Stop:     umountroot cryptdisks-early
 # Should-Start:      udev mdadm-raid lvm2
-# Should-Stop:       udev mdadm-raid lvm2
+# Should-Stop:       mdadm-raid lvm2
 # X-Start-Before:    checkfs
 # X-Stop-After:      umountfs
 # X-Interactive:     true
-- 
2.1.4

